contracts/seed/Seed.sol

```diff
@@ -137,8 +137,8 @@ contract Seed {
     ) public initializer {
         beneficiary     = _beneficiary;
         admin           = _admin;
-        successMinimum  = _successMinimumAndCap[0];
-        cap             = _successMinimumAndCap[1];
+        softCap         = _successMinimumAndCap[0];
```

Comment:
pls adjust `_successMinimumAndCap`  to the updated naming, could `softAndHardCap` or something similar

Reply:
pls do it in the `SeedFactory` contract as well

Reply:
Updated it.